tests/test_tasks/test_task_functions.py

```diff
@@ -129,6 +129,24 @@ def test_get_task(self):
             self.workdir, 'org', 'openml', 'test', "datasets", "1", "dataset.arff"
         )))
 
+    def test_get_task_lazy(self):
+        task = openml.tasks.get_task(2, download_data=False)
+        self.assertIsInstance(task, OpenMLTask)
+        self.assertTrue(os.path.exists(os.path.join(
+            self.workdir, 'org', 'openml', 'test', "tasks", "2", "task.xml",
+        )))
+
+        self.assertFalse(os.path.exists(os.path.join(
+            self.workdir, 'org', 'openml', 'test', "tasks", "2", "datasplits.arff"
+        )))
+        # Not checking datasets/ as other test cases might interfere
```

Comment:
The cache directories are unique for each unit test so this should not be an issue: https://github.com/openml/openml-python/blob/master/openml/testing.py#L59